### PR TITLE
added wstETH to terra2 chain registry

### PR DESCRIPTION
**Pull request by fluffytrex**

### CODE REVIEW

File: terra2/assetlist.json [hunk 1]
```diff
@@ -510,4 +510,52 @@
...
+  "name": "Canonical Wrapped stETH",
+  "display": "wstETH",
+  "symbol": "wstETH",
+  "traces": [
+    {
+      "type": "ibc",
+      "counterparty": {
+        "chain_name": "neutron",
+        "base_denom": "factory/neutron1ug740qrkquxzrk2hh29qrlx3sktkfml3je7juusc2te7xmvsscns0n2wry/wstETH",
+        "channel_id": "channel-25"
```
Comment (JeremyParish69) [inline terra2/assetlist.json:539]: Can we get this ibc connection registered first, please?

File: terra2/assetlist.json [hunk 2]
```diff
@@ -510,4 +510,52 @@
...
+  "base": "ibc/A356EC90DC3AE43D485514DA7260EDC7ABB5CFAA0654CE2524C739392975AD3C",
+  "name": "Canonical Wrapped stETH",
+  "display": "wstETH",
+  "symbol": "wstETH",
+  "traces": [
+    {
+      "type": "ibc",
+      "counterparty": {
+        "chain_name": "neutron",
+        "base_denom": "factory/neutron1ug740qrkquxzrk2hh29qrlx3sktkfml3je7juusc2te7xmvsscns0n2wry/wstETH",
```
Comment (JeremyParish69) [inline terra2/assetlist.json:538]: We need this asset to be registered to Neutron as well

File: terra2/assetlist.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
         },
         {
           "denom": "luna",
-          "exponent": 6
+          "exponent": 18
```
Comment (JeremyParish69) [inline terra2/assetlist.json:14]: I'm skeptical about this change to 18 and want to say it's wrong... Are you certain LUNA has exponent 18?

Comment (fluffytrex) [reply terra2/assetlist.json:14]: You are absolutely right. I meant to change line 525. The exponent of wstETH should be changed to 18. I made a mistake and initially put it at 6. When I went back to edit, I changed the wrong line.

File: terra2/assetlist.json (hunk 1 above)
Comment (fluffytrex) [reply terra2/assetlist.json:539]: I don't know what it is or how to register it... I just IBC bridge wstETH from Terra to Neutron and looked at the TX, and saw that the dst_channel was 25.

File: terra2/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply terra2/assetlist.json:538]: Another PR has just registered this asset to Neutron, so now we can reference it in traces here

File: terra2/assetlist.json (hunk 3 above)
Comment (JeremyParish69) [reply terra2/assetlist.json:14]: Ok, please make the necessary corrections

File: terra2/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply terra2/assetlist.json:539]: in the _IBC directory there are ibc connections registered. We need one for neutron<>terra2 before we can add assets that originate over these ibc connections

File: terra2/assetlist.json (hunk 3 above)
Comment (fluffytrex) [reply terra2/assetlist.json]: Corrections made.

File: terra2/assetlist.json (hunk 1 above)
Comment (fluffytrex) [reply terra2/assetlist.json:539]: Added the connection, see my most recent commit.

**Pull request merged**